rpms/SPECS/wazuh-agent.spec

```
@@ -620,6 +620,11 @@ rm -fr %{buildroot}
 
 %changelog
 * Thu Nov 03 2022 support <info@wazuh.com> - 4.4.0
```

Same, missing one line.
```suggestion
* Thu Nov 03 2022 support <info@wazuh.com> - 4.4.0
- More info: https://documentation.wazuh.com/current/release-notes/
```